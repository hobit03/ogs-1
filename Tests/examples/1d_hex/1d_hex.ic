GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
#STOP

