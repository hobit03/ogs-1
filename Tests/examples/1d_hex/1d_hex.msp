GeoSys-MSP: Material Solid Properties ------------------------------------
#SOLID_PROPERTIES
  $DENSITY
    1 2.00000e+003
#STOP


;-------------------------------------------------------

#SOIL_PROPERTIES
 2  1.000000e+000             ; dimension, (area), 
 0                            ; porosity model
 0.50000e-000  1.000000e+000 ; porosity, tortuosity
 0                            ; mobile immobile model
 0                            ; lithological component
 0                            ; maximum sorption model
 0 ; nonlinear flow parameter
 0.000000e+000 ; storativity
 0 0  1.000000e-012 ; permeability model, permeability tensor, permeabilities
 0  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000 ; k-S function
 0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000 ; p-S function
 0.2500e+000  0.00000e+000 ; mass dispersion parameters
 0.000000e+000  0.000000e+000 ; heat dispersion parameters
 2.000000e+003  0.000000e+000 ; rock density, heat capacity
 0  0.000000e+000 ; heat conductivity parameters


