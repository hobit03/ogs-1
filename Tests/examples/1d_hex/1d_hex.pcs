GeoSys-PCS: Processes ------------------------------------------------
#PROCESS
 $PCS_TYPE
  GROUNDWATER_FLOW
#STOP

