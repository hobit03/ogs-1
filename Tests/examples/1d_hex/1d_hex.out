GeoSys-OUT: Output ------------------------------------------------
#OUTPUT
 $NOD_VALUES
  HEAD
  VELOCITY_X1
  VELOCITY_Y1
  VELOCITY_Z1
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP

#OUTPUT
 $PCS_TYPE
  FLUID_MOMENTUM
 $ELE_VALUES
  VELOCITY1_X
  VELOCITY1_Y
  VELOCITY1_Z
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  86400.0
#STOP
